plugins/hydra_ray_launcher/hydra_plugins/hydra_ray_launcher/ray_remote_invoke.py

```diff
@@ -41,7 +51,7 @@ def launch_jobs(temp_job_dump_dir: str, temp_return_dump_dir: str) -> None:
             )
             runs.append(ray_obj)
 
-    _dump_job_return([ray.get(run) for run in runs], temp_return_dump_dir)
+    _dump_job_return([ray.get(run) for run in runs], temp_dir)
```

Comment:
since ray.get() is an important thing here that is actually blocking the call until the underlying function returns, I don't want to hide it inside _jump_job_return().
separate into two lines.